app/components/modals/speakerimg-cropper-modal.js

```diff
@@ -0,0 +1,42 @@
+import ModalBase from 'open-event-frontend/components/modals/modal-base';
+
+export default ModalBase.extend({
+  onVisible() {
+    let viewport = {};
+    let factor = 150;
+    const aspectRatio = this.getWithDefault('aspectRatio', [2,2]);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [A space is required after ','.](https://app.codacy.com/manual/fossasia/open-event-frontend/pullRequest?prid=4844788)